test/unit/mdc-ripple/foundation.test.js

```diff
@@ -176,6 +176,18 @@ testFoundation('#destroy removes all CSS variables', ({foundation, adapter, mock
   });
 });
 
+testFoundation('#destroy clears the activation timer if pending',
+({foundation, adapter, mockRaf}) => {
```

Comment:
I think this indentation and the presence of `adapter` when unused is throwing off our linter (check `npm run lint` in the repo; `npm run fix:js` might help)